### ISSUE TITLE
cannot preview

### ISSUE THREAD

**Issue by antoineDeroo** [HarshdeepGupta/live-html-preview#32]
Issue Type: <b>Bug</b>

I tried : 
- type "F1" + write "show side preview" + click on it
- click on "preview Available" at the bottom left (writen in the blue bar)

The "preview" window does not appear. 

Extension version: 0.3.0
VS Code version: Code 1.38.1 (b37e54c98e1a74ba89e03073e5a3761284e3ffb0, 2019-09-11T13:35:15.005Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz (6 x 3000)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.80GB (1.58GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by ghost** [comment]
Same on codium on Fedora Linux. The extension is prob broken/outdated.

**Comment by kostasx** [comment]
Can you install this updated [forked](https://marketplace.visualstudio.com/items?itemName=kostasx.live-html-previewer-v2) version and check if it works @fmlatghor ?

**Comment by ghost** [comment]
It kind of worked? The css file linked to it won't work with the index.html file.
-- 
 Securely sent with Tutanota. Get your own encrypted, ad-free mailbox: 
 https://tutanota.com


Mar 15, 2020, 18:59 by notifications@github.com:

**Comment by kostasx** [comment]
Yes, that's a limitation. You can only preview internal or inline styles at the moment.

**Comment by kangarooo** [comment]
> Can you install this updated [forked](https://marketplace.visualstudio.com/items?itemName=kostasx.live-html-previewer-v2) version and check if it works @fmlatghor ?

this one doesnt work but fork works https://marketplace.visualstudio.com/items?itemName=kostasx.live-html-previewer-v2
